src/utils/cli-utils.ts

```diff
@@ -21,7 +21,8 @@ export const copyToClipBoardUtility = (str: string): ((e: SyntheticEvent<HTMLEle
 };
 
 export function getCLISetConfigRegistry(command: string, scope: string, registryUrl: string): string {
-  return `${command} ${scope}registry ${registryUrl}`;
+  // if there is a scope defined there needs to be a ":" separator between the scope and the registry
+  return `${command} ${scope}${scope !== '' ? ':' : ''}registry ${registryUrl}`;
```

Comment:
```suggestion
  return `${command} ${scope}${!!scope ? ':' : ''}registry ${registryUrl}`;
```

Reply:
I believe registry should be translated

Reply:
I think registry cannot be translated, it is a command. It has to be in English.

Reply:
oh sure! I missed it ..thanks @juanpicado 👍